anti.md

```diff
@@ -1,43 +1,50 @@
 # Anti project
-
+Limiting Social Media Use
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+The project would serve those that feel that social media is distracting and they spend too much time doing it, but still worry about missing out on important things that happen. These people often feel pressured to be checking social media (even subconcious pressure) when they want to be more engaged with the present. With this their experience could be less distracted.
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+A language might be a good way to do this because they could query all of their social media for important updates and get them all back at once without worry about checking stuff throughout the day. But actually, a language is a bad solution because there is really only _one function_ and that could just be a button press so an app might be a better solution.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+I also feel this way! I hate feeling attached to my phone, and I came up with it both through personal experience and also through many conversations with people.
```

Comment:
Me too. As an experiment, try to go a day with a dead phone. When I broke my previous phone, the week I was without a phone was liberating and excruciating!